templates/apis/crd.go.tpl

```diff
@@ -50,6 +50,9 @@ type {{ .CRD.Kind }}Status struct {
 {{- range $column := .CRD.AdditionalPrinterColumns }}
 // +kubebuilder:printcolumn:name="{{$column.Name}}",type={{$column.Type}},JSONPath=`{{$column.JSONPath}}`
 {{- end }}
+{{- if .CRD.PrintAgeColumn }}
+// +kubebuilder:printcolumn:name="Age",type="date",priority=0,JSONPath=".metadata.creationTimestamp"
```

Comment:
Priority 0?

Reply:
That's also the default for all the builtin types (pods, services..)